core/src/replay_stage.rs

```diff
@@ -662,6 +664,12 @@ impl ReplayStage {
                     .expect("new rayon threadpool");
```

Comment:
Maybe rename this thread pool `solReplayFork` to help distinguish threads even more?

Reply:
I like it, and comes right in at the 15 character limit 🚀